src/components/reducers.js

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Ten plik nie powinien mieścić się w katalogu `components` bo nie jest komponentem. 
Jak już to `src`.